airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -448,7 +448,7 @@ def _validate_extra_conf(conf: dict[Any, Any]) -> bool:
         if conf:
             if not isinstance(conf, dict):
                 raise ValueError("'conf' argument must be a dict")
-            if any(True for k, v in conf.items() if not (v and isinstance(v, str) or isinstance(v, int))):
+            if not all((v and isinstance(v, str)) or isinstance(v, int) for v in conf.values()):
```

Comment:
```suggestion
            if not all(isinstance(v, (str,int)) for v in conf.values()):
```

Same here. I think - since we are anyhow modifying the conditions - we should fix them as well.